### DISCUSSION TITLE
Сохранить файл в поле по внешней ссылке url

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#14] [Как это сделать?] [answered]
Здравствуйте, Алексей. Не большой вопрос:
можно ли в тотуме при нажатии кнопки сохранить в поле файл, который находится по внешней ссылке типа "https://t-video.by/docs/price_ vds.pdf..."?
Спасибо

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Для того, что бы сохранить файл нам потребуется поле типа [Файл](https://docs.totum.online/fields#type-file). В него мы через [set](https://docs.totum.online/functions#fn-set) по [Кнопке](https://docs.totum.online/fields#type-button) будем класть файл.

**Код действия кнопки:**

```totum

=: set(table: $#ntn; field: 'h_file' = $filesData)
    filesData: listCreate(item: $rowFile)
        rowFile: rowCreate(field: "name" = "Загруженный файл.png"; field: "filestring" = $get)

get: getFromScript(uri: "https://totum.totum.online/fls/365_392_file.png")

```

Здесь история следующая. В поле [Файл](https://docs.totum.online/fields#type-file) всегда список, поэтому мы выполняем [listCreate](https://docs.totum.online/functions#fn-listCreate) из `row`, который имеет два ключа — `name` и `filestring`. В `filestring` мы передаем то, что получил [getFromScript](https://docs.totum.online/functions#fn-getFromScript) по скормленной ему ссылке.

Здесь пример с хедером, если сетим в строчную часть, то не забудте параметр `where`.
